tutorials/s3output/src/main/resources/META-INF/properties.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```

Comment:
The application as such is not functional as of now.
Can we mention this in the README?
Also in the README, we should specify the parameters and sample values for them as they are missing from the current properties file.